src/alt.js

```diff
@@ -454,6 +461,7 @@ function createStoreFromClass(alt, StoreModel, key, ...argsForConstructor) {
 
 class Alt {
   constructor(config = {}) {
+    this.config = config
```

Comment:
I like the pattern of being explicit and only the constructor getting the config. Then each thing that we want alt to know about can be pulled off the config. That is just my preference, I think either way is fine.  If we do go this way, I don't think we need a symbol, it would be fine as is.